src/Microsoft.Azure.ServiceBus/Primitives/ServiceBusConnection.cs

```diff
@@ -51,7 +51,7 @@ public Task CloseAsync()
 
         protected void InitializeConnection(ServiceBusConnectionStringBuilder builder)
         {
-            this.Endpoint = builder.Endpoint;
+            this.Endpoint = new Uri(builder.Endpoint);
```

Comment:
@SeanFeldman Thanks for pointing out. That needs to change to show the entire namespace. I remembered that was the reason why I had put the PR on block. 😉